packages/react-kit/src/components/productCard/ProductCard.styles.ts

```diff
@@ -156,6 +155,9 @@ export const ProductCardWrapper = styled.div<{ $isHoverDisabled: boolean }>`
   justify-content: space-between;
   flex-direction: column;
   cursor: pointer;
+  min-height: unset;
+  height: 286px;
```

Comment:
Id use this instead (change it to rem btw):
```suggestion
  min-height: 286px;
```

Reply:
It will destroy proportions, wont be perfectly like on figma ( after & before images):

<img width="412" alt="image" src="https://github.com/user-attachments/assets/c87cd2ab-a018-4da4-82b3-fa12361bfeea">
<img width="392" alt="image" src="https://github.com/user-attachments/assets/8fda384e-9c26-4dca-8155-5ce30cc68cd8">